src/test/java/org/minions/devfund/angela/WhackAMoleTest.java

```diff
@@ -0,0 +1,111 @@
+package org.minions.devfund.angela;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+
+import static junit.framework.TestCase.assertTrue;
```

Comment:
That package will be deprecated in the feature
You should use 
```java
import static org.junit.Assert.assertTrue;
```

Reply:
Done